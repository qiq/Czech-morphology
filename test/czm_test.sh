#!/usr/bin/env bash

base=$BASH_SOURCE
base=`dirname "$base"`
base=`readlink -f "$base"`'/..'
test_base=`readlink -f "$base"`

PATH="$test_base/test:$PATH"

./czm "$test_base/data/CZ100404a" <"$test_base/test/czm_test.in" >czm_test.out.tmp
diff "$test_base/test/czm_test.out" czm_test.out.tmp
#exit $?
