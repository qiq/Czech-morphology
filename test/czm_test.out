Metro_;G_;K NNNS1-----A---- NNNS4-----A---- NNNS5-----A----	metro NNNS1-----A---- NNNS4-----A---- NNNS5-----A----
jet-1_^(pohybovat_se,_ne_však_chůzí) VpNS---XR-AA---
odpoledne-1 Db-------------	odpoledne-2 NNNP1-----A---- NNNP4-----A---- NNNP5-----A---- NNNS1-----A---- NNNS2-----A---- NNNS4-----A---- NNNS5-----A----
xxxdfsdfsdf AAXXX----1A---- NNFXX-----A---- NNIS1-----A---- NNIS4-----A---- NNIXX-----A---- NNMS1-----A---- NNMXX-----A---- NNNXX-----A---8 NNXXX-----A---8	xxxdfsdfsdfy NNFP2-----A----	xxxdfsdfsdonfer Vi-S---2--A----
, Z:-------------
ala-1_,t_^(místnost_v_starořím._obydlí) NNFS3-----A---- NNFS6-----A----	ala-2_,t_^(boční_loď_v_bazilice) NNFS3-----A---- NNFS6-----A----	ala-3_,t_^(vojenská_jednotka_ve_st._Římě) NNFS3-----A---- NNFS6-----A----	ale J^-------------
ne TT-------------
v-1 RR--4---------- RR--6----------	v-3_^(označení_pomocí_písmene) NNNXX-----A----
jeden`1 ClFS4----------
. Z:-------------
